ui/load/wallet.go

```diff
@@ -40,6 +40,16 @@ func (wl *WalletLoad) SortedWalletList() []*dcrlibwallet.Wallet {
 	return wallets
 }
 
+// TicketPrice returns the current ticket price.
+func (wl *WalletLoad) TicketPrice() int64 {
+	pr, err := wl.MultiWallet.WalletsIterator().Next().TicketPrice()
+	if err != nil {
+		log.Error(err)
+		return 0
+	}
+	return pr.TicketPrice
+}
```

Comment:
This is similar to what we currently have `Multiwallet.TicketPrice()`. I dont think we need it.